.github/workflows/ci-run-test.yml

```diff
@@ -429,8 +430,10 @@ jobs:
       - name: Summarize Test Times by Individual Test
         continue-on-error: true
         run: |
-          goteststats -statistic test-time test-results/*.json | head -n 100 || \
-                      if [[ $? -eq 141 ]]; then true; else exit $?; fi
```

Comment:
`shellcheck` didn't like us using `$?` like this here.  I think it might have been wrong for the `exit $?`.